src/actions/channels.js

```diff
@@ -512,6 +512,8 @@ export function leaveChannel(channelId) {
         const channel = channels[channelId];
         const member = myMembers[channelId];
 
+        Client4.trackEvent('api', 'api_channels_leave', {channel_id: channelId});
```

Comment:
These should use the type `action` and the names should be prefixed with `action_` instead of `api_`

Like so:
https://github.com/mattermost/mattermost-redux/blob/master/src/actions/posts.js#L334